labsdb-worker/src/main/java/com/vmware/flowgate/labsdb/redis/RedisConfig.java

```diff
@@ -33,6 +35,7 @@ RedisMessageListenerContainer container(RedisConnectionFactory connectionFactory
    }
 
    @Bean
+   @Profile("!test")
```

Comment:
Any reason we add this ?

Reply:
Removed @Profile("!test"),  and embeded Redis have added.